src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -1426,6 +1427,55 @@ def test_aks_create_add_nodepool_with_motd(self, resource_group, resource_group_
         self.cmd(
             'aks delete -g {resource_group} -n {name} --yes --no-wait', checks=[self.is_empty()])
 
+    @AllowLargeResponse()
+    @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='eastus')
+    def test_aks_custom_ca_trust_flow(self, resource_group, resource_group_location):
+        aks_name = self.create_random_name('cliakstest', 16)
+        node_pool_name = self.create_random_name('c', 6)
+        node_pool_name_second = self.create_random_name('c', 6)
+        self.kwargs.update({
+            'resource_group': resource_group,
+            'name': aks_name,
+            'node_pool_name': node_pool_name,
+            'node_pool_name_second': node_pool_name_second,
+            'ssh_key_value': self.generate_ssh_keys(),
+        })
+
+        # 1. create
+        create_cmd = 'aks create --resource-group={resource_group} --name={name} ' \
+                     '--nodepool-name {node_pool_name} -c 1 ' \
+                     '--ssh-key-value={ssh_key_value} ' \
+                     '--enable-custom-ca-trust'
+        self.cmd(create_cmd, checks=[
+            self.check('provisioningState', 'Succeeded'),
+            self.check('agentPoolProfiles[0].enableCustomCaTrust', 'True')
+        ])
+
+        # 2. add nodepool
+        self.cmd('aks nodepool add '
+                 '--resource-group={resource_group} '
+                 '--cluster-name={name} '
+                 '--name={node_pool_name_second} '
+                 '--enable-custom-ca-trust',
+                 checks=[
+                     self.check('provisioningState', 'Succeeded'),
+                     self.check('enableCustomCaTrust', 'True'),
+                 ])
+
+        # 3. update nodepool
+        self.cmd('aks nodepool update '
+                 '--resource-group={resource_group} '
+                 '--cluster-name={name} '
+                 '--name={node_pool_name} '
+                 '--disable-custom-ca-trust',
+                 checks=[
+                     self.check('provisioningState', 'Succeeded'),
+                     self.check('enableCustomCaTrust', 'False'),
+                 ])
```

Comment:
Since no update code path was implemented, the [test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=55769479&view=logs&j=b162b355-d59d-5864-ce0f-0a70f12dd28b&t=a26fe913-e2be-5062-b08e-d15f1acc7ea2&l=585) would fail.

Reply:
As discussed offline, update path is temporarily removed from the test scenario.